package.json

```diff
@@ -28,6 +28,7 @@
     "deep-equal": "^2.0.5",
     "dotenv": "^16.0.1",
     "graphql": "^15.3.0",
+    "idle-tracker": "^0.1.3",
```

Comment:
The most light-weight one of these that did what I needed and had event throttling.